base/sysinfo.jl

```diff
@@ -200,7 +212,19 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     end
     println(io)
 end
+"""
+    Sys.cpu_summary()
+
+Prints a summary of CPU information, organizing and displaying aggregated data for CPUs with the same model.
 
+# Arguments
+- `io::IO`: Output stream where the summary will be printed (default is `stdout`).
+- `cpu::AbstractVector{CPUinfo}`: Vector of `CPUinfo` objects containing detailed information about each CPU (default is obtained from `cpu_info()`).
+
+# Output
```

Comment:
```suggestion
```
We don't normally have a section header.   But it's fine to start a new paragraph with more details.